findanagram.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'lib/anagrams'
+
+input_word = ARGV[0]
+text_file = ARGV[1]
```

Comment:
@zzokki81 can you please modify the program so I can give it a dictionary filename and enter any number of the words I want anagrams for. For example, `ruby findanagrams.rb dict.txt tapes pictures undress`.